otx/algorithms/common/configs/configuration_enums.py

```diff
@@ -33,3 +33,11 @@ class StorageCacheScheme(ConfigurableEnum):
     JPEG_95 = "JPEG/95"
     PNG = "PNG"
     TIFF = "TIFF"
+
+
+class BatchSizeAdaptType(ConfigurableEnum):
+    """This Enum represents the type of adapting batch size."""
+
```

Comment:
Actually, I'm not sure those words (NONE, SAFE, FULL) are appropriate; it's hard to look at them and predict how they will behave.

Reply:
Honestly, making an appropriate name, which is meaningful but not verbose, is also quite hard to me. I had some of candidates and decide it together in the meeting. I also think it could be confusing to someone, but I think description can help them to understand.

Reply:
Yes. It was already a done deal, then I guess we need to comment this out here more detail about Type.

Reply:
I agree on that. Then I'll add some comment there.

Reply:
done on [2dcb527](https://github.com/openvinotoolkit/training_extensions/pull/2119/commits/2dcb52706d932498d0863be6d596d597c82ce517)